guides/source/form_helpers.md

```diff
@@ -516,22 +562,26 @@ Outputs a select box like:
 </select>
 ```
 
-Notice that the appropriate option was automatically marked `selected="selected"`. Since this select box was bound to a model, we didn't need to specify a `:selected` argument!
+The only difference is that the selected option will be found in `params[:person][:city]` instead of `params[:city]`.
+
+Notice that the appropriate option was automatically marked `selected="selected"`. Since this select box was bound to an existing `@person` record, we didn't need to specify a `:selected` argument.
+
+### Selecting Time Zone
```

Comment:
I might be missing why this is one section above, but going to ask anyway: Would this section not fall under the Date and Time helpers (next section)?

Reply:
No good reason anymore, moved!